lib/src/Base/Func/openturns/PiecewiseLinearEvaluation.hxx

```diff
@@ -100,6 +102,9 @@ protected:
                                           const Bool isRegular);
 
 private:
+  /** Enable extrapolation */
+  Bool enableExtrapolation_;
```

Comment:
the new attribute must be saved, and loaded only if it exists in the study for backward-compatibility `if (adv.hasAttribute("enableExtrapolation_")) adv.loadAttribute ...)